server/hotrod/src/main/scala/org/infinispan/server/hotrod/package.scala

```diff
@@ -6,6 +6,8 @@ import java.util.function.{BiConsumer => J8BiConsumer}
 
 import org.infinispan.remoting.transport.Address
 
+import scala.language.implicitConversions
```

Comment:
same as above - says that `implicit` conversion functions are used intentionally